CONTRIBUTING.md

```diff
@@ -9,47 +9,50 @@ is welcome to do so.
 This document aims at establishing guidelines for contributions within
 Lethe.
 
-Lethe has a wiki which is openly accessible on : https://github.com/chaos-polymtl/lethe/wiki
+Lethe has a wiki which is openly accessible on : https://chaos-polymtl.github.io/lethe/index.html
 
 
 # How can I contribute?
 
 Contributions through code or documentation should be done through pull
 request at the official Github repository of Lethe : https://github.com/chaos-polymtl/lethe.
 We recommend that users either request access to the Lethe repository or
-create their own fork of Lethe on their own github account. This can then be
+create their own fork of Lethe on their own Github account. This can then be
 used to open pull requests
 
 # What should be the content of a pull request (PR)?
 
 A pull request should contain the following elements:
 
 - A brief title (less than 60 characters) that describes the goal of the pull
-requests.
+request.
 - A detailed description of the content added by the pull request:
-i) If the PR adds a new feature: The feature should be documented, tests (with
-  a unit tests and/or an application test) and if the feature adds or alters
-  parameters in the input file, the appropriate section of the wiki should
-  be updated accordingly.
-ii) If the pull requests corrects a bug: the source of the bug should be
-explained and the way it was identified should be briefy described. A unit
-test or application test that reproduces the bug should be added.
+  * If the PR adds a new feature:\
+    The feature should be documented, tested (with unit tests and/or application tests) 
+    and if the feature adds or alters parameters in the input file, the appropriate section 
+    of the documentation should be updated accordingly.
+  * If the pull request corrects a bug:\
+    the source of the bug should be explained and the way it was identified should be briefly described. 
+    A unit test or application test that reproduces the bug should be added.
+
+See [this page](https://chaos-polymtl.github.io/lethe/documentation/contributing.html#pull-requests) for more information about the pull request process.
+
```

Comment:
Unrelated:
This is maybe not for this PR and should be added to the "Miscellaneous" subsection of the PR, but a check should be done to make sure that all information here correspond to what's on the online documentation and vice-versa.
For example, the 60 char rule is not mentioned in the online doc and in this file the `prmindent` is not mentioned. However, since you link the doc here, you don't need to put all the info that's there here, but there is no link from the doc to here so the other way is the still a problem.

Reply:
Actually, if the info is somewhere, it should not be somewhere else also. Meaning, we should refer to the doc only. 
Let see what Bruno has to say about that

Reply:
I remember that there was an issue regarding the fact that when the documentation was down (during the time doxygen was added) it was not possible to know how to locally compile from the documentation online. Other than that (how to locally compile documentation), I think you're right and we could leave the rest online.